cranelift/native/Cargo.toml

```diff
@@ -21,6 +21,8 @@ libc = "0.2.95"
 default = ["std"]
 std = ["cranelift-codegen/std"]
 core = ["cranelift-codegen/core"]
+riscv64=[]
```

Comment:
Are we using this feature flag anywhere? I can't see it.

Reply:
this is used.
![image](https://user-images.githubusercontent.com/96557710/192257511-76defd6e-5c74-4e38-b62b-af9b3e7141ec.png)

Reply:
That one seems to be the one on the `cranelift-codegen` crate, instead of the `cranelift-native`. For example we don't have the `x86`, `arm64` or `s390x` features in this crate.

Reply:
@afonso360  sorry, I forget to remove it.

Reply:
deleted now.